components/Board/Card.jsx

```diff
@@ -0,0 +1,34 @@
+import { Icon } from '@/components'
+import Image from 'next/image'
+import styles from '@/styles/Board.module.css'
+
+const Card = (props) => {
```

Comment:
Please move this file to a Card folder and add it to the index

Reply:
Done